pyproject.toml

```diff
@@ -18,6 +18,7 @@ dependencies = [
     "pydantic>=2.5.2",
     "python-dotenv>=1.0.1",
     "ssb-dash-components>=0.9.1",
+    "ssb-datadoc-model==7.0.2",
```

Comment:
We don't really want to keep this here since it's not a direct dependency of ours, we should fix the dependency specification in ´dapla-toolbelt-metadata´ if it's still an issue.